pkgs/games/frogatto/default.nix

```diff
@@ -15,9 +15,9 @@ let
     genericName = "frogatto";
     categories = "Application;Game;ArcadeGame;";
   };
+  version = "unstable-2018-12-18";
```

Comment:
:+1:  Yeah I don't think `buildEnv` is set up yet too care about `pname` and `version`.